src/main/java/eu/dissco/core/digitalmediaobjectprocessor/service/HandleService.java

```diff
@@ -30,11 +30,12 @@ public class HandleService {
 
   private static final String PREFIX = "20.5000.1025/";
   private final Random random;
-  private final char[] symbols = "ABCDEFGHJKLMNPQRSTUVWXYZ1234567890".toCharArray();
+  private final char[] symbols = "ABCDEFGHJKLMNPQRSTVWXYZ1234567890".toCharArray();
```

Comment:
Oh yeah this came out of a test. In the specimen we apparently removed the “U”, I think because the swear word thing. However, in the digital media it was still there. I assumed the specimen was correct because most focus went into that one. I just check the handle API the "U" is still in there, maybe I need to change the specimen processing service